src/applications/facility-locator/tests/00-required.testcafe.spec.js

```diff
@@ -0,0 +1,21 @@
+import { Selector } from 'testcafe';
+
+import {
+  BASE_URL,
+  axeCheck,
+} from '../../../platform/testing/e2e/testcafe/helpers';
+
+fixture('Facility Locator').page(`${BASE_URL}/find-locations/`);
+
+test('Search facility and view details', async t => {
```

Comment:
Does this `test` function come from the `runner.run()` from `run-testcafe.js`?

Reply:
I'm actually not sure, but I don't think it's explicitly coming from `runner.run()`. It's just [how TestCafe tests are declared](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#tests).

Reply:
Right, I think there's some internal whatnots. Like, that `runner` grabs all the test files that match the glob string we pass it, and runs all the tests it finds, somehow making `test` a global function that can be called like this. (I could just read the source code, I know, I know. :stuck_out_tongue_closed_eyes:)

Anyhow, I was just confused for a bit because it's similar to Jest's `test`, but...this doesn't use Jest.